synapse/lib/storm.py

```diff
@@ -4994,6 +4994,8 @@ def getArgParser(self):
                           help='Include newly scraped nodes in the output')
         pars.add_argument('--skiprefang', dest='dorefang', default=True, action='store_false',
                           help='Do not remove de-fanging from text before scraping')
+        pars.add_argument('--forms', default=[], nargs='*',
```

Comment:
Not sure this can be an nargs option with an nargs value. How does the parser know the difference when you specify scraped string values?

Reply:
might be better as a single list arg though, i can change it to that